README.md

```diff
@@ -254,18 +268,14 @@ Driver=/usr/lib64/libmyodbc8a.so
 UsageCount=1
 ```
 
-## Report an issue
-
-To report an issue, provide following information:
-
-- The command that you ran and the associated output as shown on your screen by using the standard Vertica SQL client `vsql`.
-- Vertica version: `SELECT VERSION();`.
-- `DBLINK` library metadata by running statement the following as dbadmin:
-   ```
-   => SELECT * FROM USER_LIBRARIES WHERE lib_name = 'ldblink';
-   ```
-- Attach the following ODBC configuration files:
-	- `odbc.ini` (please remove passwords or other confidential information)
-	- `odbcinst.ini`
-- ODBC Driver Manager version and configuration. For example, with unixODBC, the output of the command `odbcinst -j`.
-- ODBC traces obtained while running the command (see 1.). To enable the ODBC traces you have to set `Trace = on` in `odbcinst.ini`. Do not forget to switch ODBC tracing off at the end.
+## How to report an issue
```

Comment:
Should we ask them to describe the issues in its dedicated page and to label it correctly? https://github.com/vertica/dblink/issues

We can also ask them to write any questions in the Discussion page:
https://github.com/vertica/dblink/discussions/categories/q-a

Reply:
Oops.  This was a merge mistake.  I undid the change made by @rs-vertica.

I added the links.